azure/src/components/StickyNavbar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import '../styles/StickyNavbar.css';
+
+const StickyNavbar = ({children}) => {
+  window.addEventListener('scroll', () => {
```

Comment:
EventListeners should be added on component mount and should be removed on component unmount. Otherwise Li sterns will get piled up and the application will stall.